lib/mongoose_push_web/schemas/request/send_notification/deep.ex

```diff
@@ -0,0 +1,43 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Request.SendNotification.Deep",
+    description: "Push notification request schema",
+    type: :object,
+    properties: %{
+      service: %Schema{
+        type: :string,
+        description: "Push notification service",
+        format: :string,
+        enum: ["fcm", "apns"]
+      },
+      mode: %Schema{type: :string, enum: ["prod", "dev"]},
+      priority: %Schema{type: :string, enum: ["normal", "high"]},
```

Comment:
It would be nice to say in description that default is dependent on service used.